src/cli/repo/repo.go

```diff
@@ -170,3 +175,20 @@ func getMaintenanceType(t string) (repository.MaintenanceType, error) {
 
 	return res, nil
 }
+
+// The repo update subcommand.
+// `corso repo update <repository> [<flag>...]`
+func updateCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   updateCommand,
```

Comment:
This command is a fine addition, but it also opens up a lot of (user-ended) questions about what updates are supported.  We have a lot of repo configuration controls in initialize that can only used via the init process.  Retention and caching and etc.  And update would normally imply that we do, or will, support changing those configurations with the use of this command.  Only releasing with support for the passphrase is fine, but it should also be documented on the website what updates are and are not supported, along with guides on how to use them.

I'm also pondering the preference of a flag instead of a subcommand.  `corso repo update s3 --update-passphrase foo` is fine, but I worry that if we ever add one flag that isn't simultaneously compatible with another update flag then we'll have a handful of a time ensuring behavior.  Maybe `corso repo update s3 passphrase` is better for the long term?

Pinging @gmatev and @ashmrtn for this conversation.

Reply:
Any decision on how to move forward on this?

Reply:
Since this is mostly a passthrough to Kopia, why not actually follow the same pattern https://kopia.io/docs/reference/command-line/common/repository-change-password/? 
`corso repo update-passphrase --new-passphrase` as @ashmrtn suggested?

Reply:
done